### PR TITLE
fix: Upload session commit return None on 202

### PR DESCRIPTION
**Pull request by lukaszsocha2**
Closes: SDK-2089
Fixes: #715

### CODE REVIEW

**Review by antusus — DISMISSED** [review]
👍

File: boxsdk/object/upload_session.py [hunk 1]
```diff
@@ -126,13 +127,18 @@ def commit(
...
-            return None
+
+        try:
+            response = self._session.post(
+                self.get_url('commit'),
+                headers=headers,
+                data=json.dumps(body),
+            )
+        except BoxAPIException as box_api_exc:
+            if box_api_exc.status == 202:
```
Comment (kainjow) [inline boxsdk/object/upload_session.py:138]: could use this from `http` module:
```suggestion
            if box_api_exc.status == HTTPStatus.ACCEPTED:
```

Comment (lukaszsocha2) [reply boxsdk/object/upload_session.py:138]: good point

**Review by antusus — APPROVED** [review]
👍

**Pull request merged**